vital/types/camera_rpc.cxx

```diff
@@ -90,9 +86,9 @@ ::back_project( const vector_2d& image_pt, double elev )
   vector_2d b;
 
   A.block<1, 2>( 0, 0 ) = rpc_coeffs().block<1, 2>( 0, 1 )
-                          - norm_pt[0] * rpc_coeffs().block<1, 2>( 1, 1 );
-  A.block<1, 2>( 1, 0 ) = rpc_coeffs().block<1, 2>( 2, 1 )
-                          - norm_pt[1] * rpc_coeffs().block<1, 2>( 3, 1 );
+                           - norm_pt[0] * rpc_coeffs().block<1, 2>( 1, 1 );
+  A.block<1, 2>( 1, 0 ) = rpc_coeffs().block<1, 2>( 2, 1 );
+                           - ( norm_pt[1] * rpc_coeffs().block<1, 2>( 3, 1 ) );
```

Comment:
Minor nitpick: you added parentheses in this line, but not for the previous block.  I would keep lines 89 and 91 consistent.

Reply:
Fixed